### DISCUSSION TITLE
Can batch routing be supported?

### DISCUSSION THREAD

**Discussion by dickyangyi** [postalserver/postal#3025] [Help with using Postal]
I have about 1000 mailboxes that need to be imported and routed to other related mailboxes, one by one, and I can't find a better way to batch edit them

**Comment by willpower232** [comment]
Postal doesn't really have the concept of mailboxes so not fully sure what you're trying to accomplish.

If you set up a few, then you can examine the database and copy-paste rows until you reach your 1000 addresses which is probably the fastest way.

**Reply by dickyangyi** [reply]
You are right, maybe I did not express it clearly. I need to route mailboxes in batches, for example, a@a.com to b@b.com. In this operation, I need 1000 operations. Thank you very much for your reply

**Reply by vpulshah** [reply]
Hi,

Have you setup all 1000 Domain in postal ?

I can help you to rotate mailboxes with sending limit.

Connect me on Email - vpulshah@gmail.com

Thanks
